.env

```diff
@@ -2,6 +2,8 @@
 # Set variables here that may be different on each deployment target of the app, e.g. development, staging, production.
 # https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
 
+DEFAULT_FORMAT=xml
```

Comment:
Just asking: what does this option define? Which default format will be XML? Thanks.

Reply:
Not sure why this is here... :) my bad.